test/components/kubernetes/namespaces_labels_test.go

```diff
@@ -46,6 +46,10 @@ func TestAllNamespacesHaveNameLabels(t *testing.T) {
 			continue
 		}
 
+		if testutil.IsSystemNamespace(name) {
+			continue
+		}
```

Comment:
How about `util.IsPlatformSupported(t, []util.Platform{util.PlatformAKS}) && name == "calico-system"` ? This is very specific and whitelists only this specific case, so it will remain sensitive for any other unexpected changes.

I think `IsSystemNamespace` name is quite confusing, as I'd expect this function to match `kube-system`, `kube-public`.

Reply:
Yeah we have one more confusing name `IsUserNamespace` which includes namespaces like `kube-system`, `kube-public`, etc: https://github.com/kinvolk/lokomotive/blob/37a01bf709efc106a9f50528ca61231bdbcd56a2/test/components/util/util.go#L539-L548

Reply:
Feel free to rename that as well for something more appropriate :)